frontend/lib/src/components/widgets/ChatInput/ChatInput.tsx

```diff
@@ -61,7 +61,10 @@ const isEnterKeyPressed = (
   // https://bugs.chromium.org/p/chromium/issues/detail?id=79407
 
   const { keyCode, key } = event
-  return key === "Enter" || keyCode === 13 || keyCode === 10
+  return (
+      (key === "Enter" || keyCode === 13 || keyCode === 10) &&
+      !event.nativeEvent.isComposing
```

Comment:
one small request (same comment in the other file changed by this PR): It'd be nice to have a short comment here describing why the `!event.nativeEvent.isComposing` is needed.